Tests/Tests/STPPaymentHandlerFunctionalTest.m

```diff
@@ -29,45 +29,8 @@ - (void)setUp {
     [STPAPIClient sharedClient].publishableKey = STPTestingDefaultPublishableKey;
 }
 
-- (void)testAlipayOpensNativeURLAndCancels {
-    
-    __block NSString *clientSecret = @"pi_1GiohpFY0qyl6XeWw09oKwWi_secret_Co4Etlq8YhmB6p07LQTP1Yklg";
-    id applicationMock = OCMClassMock([UIApplication class]);
-    OCMStub([applicationMock sharedApplication]).andReturn(applicationMock);
-
-    OCMStub([applicationMock openURL:[OCMArg any]
-                             options:[OCMArg any]
-                   completionHandler:([OCMArg invokeBlockWithArgs:@YES, nil])]).andDo(^(__unused NSInvocation *_) {
-        // Simulate the Alipay app opening, followed by the user returning back to the app
-        dispatch_async(dispatch_get_main_queue(), ^{
-            [[NSNotificationCenter defaultCenter] postNotification:[NSNotification notificationWithName:UIApplicationWillEnterForegroundNotification object:nil]];
-        });
-    });
-    
-    // ...should not present anything
-    OCMReject([self.presentingViewController presentViewController:[OCMArg any] animated:YES completion:[OCMArg any]]);
-    
-    STPPaymentIntentParams *confirmParams = [[STPPaymentIntentParams alloc] initWithClientSecret:clientSecret];
-    confirmParams.paymentMethodOptions = [STPConfirmPaymentMethodOptions new];
-    confirmParams.paymentMethodOptions.alipayOptions = [STPConfirmAlipayOptions new];
-    confirmParams.paymentMethodParams = [STPPaymentMethodParams paramsWithAlipay:[STPPaymentMethodAlipayParams new] billingDetails:nil metadata:nil];
-    confirmParams.returnURL = @"foo://bar";
-    
-    XCTestExpectation *e = [self expectationWithDescription:@""];
-    [[STPPaymentHandler sharedHandler] confirmPayment:confirmParams withAuthenticationContext:self completion:^(STPPaymentHandlerActionStatus status, STPPaymentIntent * __unused paymentIntent, __unused NSError * _Nullable error) {
-        // ...should attempt to open the native URL (ie the alipay app)
-        OCMVerify([applicationMock openURL:[OCMArg any]
-                                   options:[OCMArg any]
-                         completionHandler:[OCMArg isNotNil]]);
-        // ...and since we didn't actually authenticate, the final state is canceled
-        XCTAssertEqual(status, STPPaymentHandlerActionStatusCanceled);
-
-        [e fulfill];
-    }];
-    [self waitForExpectationsWithTimeout:2 handler:nil];
-}
-
-- (void)testAlipayOpensWebviewAfterNativeURLFails {
+// N.B. Test mode alipay PaymentIntent's never have a native redirect so we can't test that here
```

Comment:
How'd this pass before?

Reply:
@csabol-stripe wrote this comment, I'm not sure